### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by okadahiroyuki** [HumanSupportRobot/QandA#104] [Q&A]
We are getting more and more familiar with the HSR and want to change the speed of motions. However, we might not be able to do so without alternating the docker image on the robot. Is it possible to get the sources of the image so that we can create a modified version? Or is there another way to access the motors directly without interfering with the TMC software?

**Comment by TomohiroOno33** [comment]
Which speeds do you want to improve?
Have you tried the following settings?
https://docs.hsr.io/hsr_customization_manual_en/configuration.html#speeding-up-each-joint
